docs/minikube.md

```diff
@@ -56,51 +58,126 @@ default web browser.
         deployment.apps/percona-server-mysql-operator created
         ```
 
-3. Because minikube runs locally, the Operator will be unable to spread the
-    cluster on several nodes. Therefore default ``deploy/cr.yaml`` file should
-    be edited to adapt the Operator for the installation on a single computer.
-    Set **all occasions** of the `antiAffinityTopologyKey` key to `"none"`.
-    When done, apply the updated ``deploy/cr.yaml`` file with the following
-    command:
-
-    ```bash
-    kubectl apply -f deploy/cr.yaml
+## Configure and deploy your MySQL cluster
+
+Since Minikube runs on a single node, you need to adjust the default configuration. The Operator normally spreads pods across multiple nodes, but Minikube only has one node available.
+
+Edit the `deploy/cr.yaml` file and set **all occurrences** of the `antiAffinityTopologyKey` key to `"none"`. This allows the Operator to run all pods on a single node.
+
+Here's the example of the modified `deploy/cr.yaml` file:
```

Comment:
@nastena1606 maybe we can refer the file from the repo instead of having it hardcoded here, WDYT?

Reply:
Yes, will update the doc accordingly.